dashboard/app/models/script.rb

```diff
@@ -333,8 +333,9 @@ def self.lesson_extras_script_ids
     @@lesson_extras_script_ids ||= all_scripts.select(&:lesson_extras_available?).pluck(:id)
   end
 
-  def self.maker_units
-    @@maker_units ||= visible_units.select(&:is_maker_unit?)
+  def self.maker_units(user)
+    return_units = @@maker_units ||= visible_units.select(&:is_maker_unit?)
+    return_units + all_scripts.select {|s| s.has_pilot_access?(user) && s.is_maker_unit?}
```

Comment:
This is minor-ish, but I think `is_maker_unit?` is a simpler check and should probably go first.

Reply:
Flipped- thanks!